src/EventGridEmulator.sln

```diff
@@ -29,18 +29,26 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{B77D5E31-0072-441F-8F88-081E0219B7DF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{B77D5E31-0072-441F-8F88-081E0219B7DF}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{B77D5E31-0072-441F-8F88-081E0219B7DF}.NoVariablesDefault|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
Why did we add this one? I mean the NoVariablesDefault, wasn't debug and release enough?